applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -114,77 +128,68 @@ def __init__(self, main_model_part, custom_settings):
         self.main_model_part = main_model_part
         self.print_on_rank_zero("::[MechanicalSolver]:: ", "Construction finished")
 
+        # Set if the analysis is restarted
+        load_restart = self.settings["restart_settings"]["load_restart"].GetBool()
+        self.main_model_part.ProcessInfo[KratosMultiphysics.IS_RESTARTED] = load_restart
```

Comment:
No flag?, maybe is a good idea to use a flag here

Reply:
what do you mean?
This is a true or false flag...

Reply:
Looks like a variable

Reply:
It is a variable 
![imagen](https://user-images.githubusercontent.com/19991680/37037954-c0e7498c-2153-11e8-919a-bd95445aecfb.png)

Reply:
These are the flags

![imagen](https://user-images.githubusercontent.com/19991680/37038009-e80a45d2-2153-11e8-9d50-1549a29b9049.png)


Before adding one it is supposed to be discussed. But I think this flag is relevant enough to be deined

Reply:
ah now I got what you mean
I think this could be changed in the future, afaik it is also being used elsewhere, i.e. I did not come up with this

Reply:
Just suggesting @KratosMultiphysics/technical-committee about adding a RESTART flag, instead of using the variable IS_RESTARTED